schemas/collector-report.md

```diff
@@ -0,0 +1,67 @@
+# The collector report JSON schema
+
+This document details the structure of reports collected by https://github.com/foolip/mdn-bcd-collector and stored in https://github.com/foolip/mdn-bcd-results.
```

Comment:
The Gooborg Studios version is what is linked throughout this repository, so we should probably keep with that trend to reduce confusion:

```suggestion
This document details the structure of reports collected by https://github.com/GooborgStudios/mdn-bcd-collector and stored in https://github.com/GooborgStudios/mdn-bcd-results.
```

(There is, of course, a bit of self-interest reasoning involved there... :P)